### DISCUSSION TITLE
Application error on getting random quotes by tags

### DISCUSSION THREAD

**Discussion by Anish-Chanda** [lukePeavey/quotable#137] [Report Downtime]
I am getting an application error only when i request quotes based on tags, this used to work well yesterday and if im not wrong i think it worked in the morning (around 9am ist) too.

**To Reproduce**
1. Open an API client (Postman or Chrome)
2. Make the following API request > https://api.quotable.io/random?tags=history%7Ccivil-rights
3. (you get an application error but you don't get one on this request > https://api.quotable.io/random

**Comment by omeedf** [comment]
I'm experiencing the same error for all API requests

**Comment by damylolae** [comment]
> I am getting an application error only when i request quotes based on tags, this used to work well yesterday and if im not wrong i think it worked in the morning (around 9am ist) too.
> 
> **To Reproduce**
> 
> 1. Open an API client (Postman or Chrome)
> 2. Make the following API request > https://api.quotable.io/random?tags=history%7Ccivil-rights
> 3. (you get an application error but you don't get one on this request > https://api.quotable.io/random

I think their server is down, nothing seems working

**Comment by lukePeavey** [comment]
Yes, it is.

**Comment by lukePeavey** [comment]
Working on it

**Comment by lukePeavey** [comment]
Fixed. Thanks for reporting.

**Comment by lukePeavey** [comment]
Moving this to discussions area.
